apps/microtvm/reference-vm/zephyr/base-box/base_box_test.sh

```diff
@@ -37,3 +37,5 @@ if [ $board == "stm32f746g_disco" ] || [ $board == "nucleo_f746zg" ]; then
 else
     pytest tests/micro/zephyr/test_zephyr_aot.py --zephyr-board=${board}
 fi
+
+pytest tests/micro/zephyr/test_zephyr_armv7m.py --zephyr-board=${board}
```

Comment:
could you move this to tests/scripts/task_python_microtvm.sh? This is just the base-box pre-release test, where we just want to do a smoke test to make sure that Zephyr is working with attached hardware.

Reply:
It seems moving to tests/scripts/task_python_microtvm.sh make no sense. The only supported platform there is qemu_x86 for now. We don't run our test with it. And mps2_an521 disabled.
BTW, all tests in tests/micro/zephyr must be called, isn't it?

Reply:
we discussed offline and have decided to proceed with local testing in this PR. in a follow-on, we will add support for Corstone 300 tests.